backend/src/User/user.controller.ts

```diff
@@ -75,8 +75,27 @@ class Controller {
             passport.authenticate('jwt', { session: false }),
             async (req: Request, res: Response) => {
                 const { name, email, role, password } = req.body;
-                const result = await this.userService.createNewUser(name, role, email, password);
-                res.json(result);
+                var message;
+
+                try {
+                    const result = await this.userService.createNewUser(
+                        name,
+                        role,
+                        email,
+                        password
+                    );
+                    message = {
+                        id: result,
+                        message: 'User was created successfully.'
+                    };
+                } catch (e) {
+                    message = {
+                        message: e as string
+                    };
+                    res.status(400);
```

Comment:
```suggestion
                    message = {
                        status: false,
                        error: e as string
                    };
```

The front end does not read 400 errors, just pass the error in the error field.

![image](https://user-images.githubusercontent.com/43256461/111314469-d3394080-8637-11eb-8a20-7b930e086358.png)

Reply:
Done!